app/models/queries/GalleryQueries.py

```diff
@@ -67,4 +68,33 @@ def update(gid, title, open_date, close_date, description, banner):
         return gid
     except Exception as e:
         print (e)
-    return None
\ No newline at end of file
+    return None
+    
+def getstatus(gid):
```

Comment:
pip8 is based on underscores, two words should be separated using underscores, so get_status not getstatus or getStatus

Reply:
Noted